models/src/test/java/com/rudderstack/models/RudderServerConfigParseTest.kt

```diff
@@ -419,7 +423,7 @@ open class RudderServerConfigParseTest {
     }
 
     @Test
-    fun testDeserialization() {
+    fun testJsonDeserialization() {
```

Comment:
:idea Just a small patch to help you organise better your integration tests:
[feat__mock_response_and_json_resources.patch](https://github.com/rudderlabs/rudder-sdk-android/files/15461730/feat__mock_response_and_json_resources.patch)

Reply:
awesome. Pushing it. 💯